modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -15,7 +15,7 @@ In the table, features are marked with the following statuses:
 |*OpenShift GitOps* 6+|*Component Versions*|*OpenShift Versions*|*Support status*
 
 |*Version*|*kam*|*Helm*|*Kustomize*|*Argo CD*|*ApplicationSet*|*Dex*||
-|1.3.0|0.0.40|3.6.0|4.2.0|2.1.2|0.2.0|2.28.0|4.6-4.9|GA
+|1.3.0|0.0.40|3.6.0|4.2.0|2.1.8|0.2.0|2.30.0|4.6-4.9|GA
```

Comment:
Dont we update patch releases here ?

1.3.0 does not support Dex 2.30.0, it is actually supported by 1.3.2. To be more clear I think we should update the dependency matrix for z-stream releases as well.